sections/futures/Trade/TransferIsolatedMarginModal.tsx

```diff
@@ -51,6 +62,16 @@ const TransferIsolatedMarginModal: React.FC<Props> = ({ onDismiss, defaultTab })
 		position?.accessibleMargin,
 	]);
 
+	const isLowBalance = useMemo(() => susdBalance.gt(zeroBN) && susdBalance.lt(minDeposit), [
+		susdBalance,
+		minDeposit,
+	]);
+
+	const isNoBalance = useMemo(() => susdBalance.eq(zeroBN), [susdBalance]);
```

Comment:
I think it might be nice to merge these two into a single memo `balanceStatus` with `'low_balance' | 'no_balance' | 'high_balance'`

Reply:
Agree. Will do it.